ddl/ddl_worker.go

```diff
@@ -319,6 +305,24 @@ func (d *ddl) addBatchDDLJobs(tasks []*limitJobTask) {
 	logutil.BgLogger().Info("[ddl] add DDL jobs", zap.Int("batch count", len(tasks)), zap.String("jobs", jobs))
 }
 
+func resolveJobListKey(tp model.ActionType) meta.JobListKeyType {
+	if tp == model.ActionAddIndex ||
+		tp == model.ActionAddPrimaryKey ||
+		tp == model.ActionModifyColumn {
+		return meta.AddIndexJobListKey
+	}
+	return meta.DefaultJobListKey
+}
+
+func resolveJobListKeyNeg(tp model.ActionType) meta.JobListKeyType {
```

Comment:
`resolveJobListKeyNeg` is a bad name. I don't know what it means till I read the code. `resolveOtherJobListKey` maybe a better name?